src/common/utils/api/api.ts

```diff
@@ -275,7 +277,14 @@ export default {
     return {
       name: configResponse.name,
       resourceState: {
-        options: resourceStateOptions,
+        options: [
+          ...resourceStateOptions,
+          // TODO: This needs to be returned from the server
+          {
+            label: { fi: 'Muu, mikä?', sv: 'Muu, mikä', en: 'Muu, mikä?' },
+            value: ResourceState.OTHER,
+          },
+        ],
```

Comment:
The API data model does not support this kind of free-form state. And probably never will because the state has to be an enumeration.
I think we need to handle this in the client if the state is open and the opening has a title then it is other.
What we can do later on is to add more common enumerations to API.